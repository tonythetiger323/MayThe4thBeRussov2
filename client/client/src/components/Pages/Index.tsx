import * as React from 'react'
import Page from '../Layouts/Page'
import Typography from '@material-ui/core/Typography'

class Index extends React.Component {
  render() {
    return (
      <div>
        <Page>
          <Typography variant='h2'>May the 4th be Russo</Typography>
        </Page>
      </div>
    )
  }
}

export default Index
