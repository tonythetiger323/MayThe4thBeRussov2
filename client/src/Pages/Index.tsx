import * as React from 'react'
import PhotoStepper from '../Components/PhotoStepper'

class Index extends React.Component {
  render() {
    return (
      <div>
        <PhotoStepper />
      </div>
    )
  }
}

export default Index
