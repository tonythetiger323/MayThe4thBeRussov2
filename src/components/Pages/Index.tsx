import * as React from 'react'
import Page from '../Layouts/Page'
import Typography from '@material-ui/core/Typography'
import PhotoStepper from '../PhotoStepper'
import { withStyles, Theme } from '@material-ui/core/styles'

const styles = (theme: Theme) => ({
  stepper: {
    height: '50px',
    width: '300px',
    opacity: 0.0
  }
})

interface IndexProps {
  classes: {
    stepper: string
  }

  propTypes: any
}
class Index extends React.Component<IndexProps, any> {
  constructor(props: IndexProps) {
    super(props)
  }

  render() {
    const style = {
      fontFamily: 'Arvo'
    }
    const { classes } = this.props
    return (
      <div>
        <Page>
          <Typography style={style} variant='h2'>
            May the 4th be Russo
          </Typography>
          <PhotoStepper className={classes.stepper} />
        </Page>
      </div>
    )
  }
}

export default withStyles(styles)(Index)
