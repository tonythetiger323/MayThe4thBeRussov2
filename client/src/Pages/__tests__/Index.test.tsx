import * as React from 'react'
import { shallow } from 'enzyme'
import Index from '../Index'

describe('Index/Home Page', () => {
  let wrapper: any
  beforeEach(() => (wrapper = shallow(<Index />)))

  it('should render correctly', () => {
    expect(wrapper).toMatchSnapshot()
  })
})
