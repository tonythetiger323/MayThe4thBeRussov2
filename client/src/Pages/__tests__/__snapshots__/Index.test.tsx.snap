// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Index/Home Page should render correctly 1`] = `ShallowWrapper {}`;
